packages/components/src/components/AssetLogo/AssetLogo.tsx

```diff
@@ -39,7 +41,7 @@ const Logo = styled.img<{ $size: number; $isVisible: boolean }>(
     `,
 );
 
-const getAssetLogoUrl = (fileName: string, quality?: '@2x') =>
+const getAssetLogoUrl = (fileName?: string, quality?: '@2x') =>
```

Comment:
```suggestion
const getAssetLogoUrl = (fileName?: string, quality?: '@2x') =>
```
This is going to build wrong url in case file name is undefined